scripts/superdesk-archive/index.js

```diff
@@ -1,3 +1,7 @@
+//styles
+import './styles/related-item.less';
```

Comment:
Would it serve better de-coupling if `relative-item-widget` would import its own styles?